docs/api/liquidstaking.md

```diff
@@ -0,0 +1,476 @@
+        "validator_address": "cantovaloper1xjlslz2vl7v6gu807fmfw8ae7726q9pf84kzqs",
+        "provider_address": "canto1xjlslz2vl7v6gu807fmfw8ae7726q9pf9t3x34",
+        "fee_rate": "0.100000000000000000",
+        "chunk_id": "0",
+        "status": "INSURANCE_STATUS_UNPAIRED"
+      },
+      "derived_address": "canto1hk5wgk3js5uqymxppawk87tv0j0fnc3pefcex4",
+      "fee_pool_address": "canto1a3f65vrngauvsj066067qsjh068hgxezpdr6rg"
+    },
+    {
+      "insurance": {
+        "id": "3",
+        "validator_address": "cantovaloper1xjlslz2vl7v6gu807fmfw8ae7726q9pf84kzqs",
+        "provider_address": "canto1xjlslz2vl7v6gu807fmfw8ae7726q9pf9t3x34",
+        "fee_rate": "0.100000000000000000",
+        "chunk_id": "0",
+        "status": "INSURANCE_STATUS_UNPAIRED"
+      },
+      "derived_address": "canto1yqg5xesskfhmzwdn3gaas6faz6d0yjwd34rrct",
+      "fee_pool_address": "canto10m6pl6am95swkaa480789y2wqlmpyqang9uxeh"
+    },
+    {
+      "insurance": {
+        "id": "4",
+        "validator_address": "cantovaloper1xjlslz2vl7v6gu807fmfw8ae7726q9pf84kzqs",
+        "provider_address": "canto1xjlslz2vl7v6gu807fmfw8ae7726q9pf9t3x34",
+        "fee_rate": "0.100000000000000000",
+        "chunk_id": "1",
+        "status": "INSURANCE_STATUS_PAIRED"
+      },
+      "derived_address": "canto1my633g6sqx9fr4szzxuj70zutmsd78zymhv5kf",
+      "fee_pool_address": "canto1sdl4z9y8x59979qjx8ut9zyndsux9sld0s6kcv"
+    },
+    {
+      "insurance": {
+        "id": "5",
+        "validator_address": "cantovaloper1xjlslz2vl7v6gu807fmfw8ae7726q9pf84kzqs",
+        "provider_address": "canto1xjlslz2vl7v6gu807fmfw8ae7726q9pf9t3x34",
+        "fee_rate": "0.100000000000000000",
+        "chunk_id": "0",
+        "status": "INSURANCE_STATUS_PAIRING"
+      },
+      "derived_address": "canto10fhsthfzmwcfjhqulyy8w0r2fqa9q5s6a88sz4",
+      "fee_pool_address": "canto1mfkktq2nj4dcc9ypkfengcpjufntk9eak0atc3"
+    },
+    {
+      "insurance": {
+        "id": "6",
+        "validator_address": "cantovaloper1xjlslz2vl7v6gu807fmfw8ae7726q9pf84kzqs",
+        "provider_address": "canto1xjlslz2vl7v6gu807fmfw8ae7726q9pf9t3x34",
+        "fee_rate": "0.100000000000000000",
+        "chunk_id": "0",
+        "status": "INSURANCE_STATUS_PAIRING"
+      },
+      "derived_address": "canto1nwhkqs073xj7sx3tq7yp7sxkqa6n4fawcy5fxf",
+      "fee_pool_address": "canto1tjqa0tc9dxv7f23szzj3m9rn88uzegf5d0ks9n"
+    },
+    {
+      "insurance": {
+        "id": "7",
+        "validator_address": "cantovaloper1xjlslz2vl7v6gu807fmfw8ae7726q9pf84kzqs",
+        "provider_address": "canto1xjlslz2vl7v6gu807fmfw8ae7726q9pf9t3x34",
+        "fee_rate": "0.010000000000000000",
+        "chunk_id": "2",
+        "status": "INSURANCE_STATUS_PAIRED"
+      },
+      "derived_address": "canto1dcd7gu8s5xez9hadt8sum0lnvfz93ntfgtq04q",
+      "fee_pool_address": "canto1v9cv3cxst0sxrz92mptx6fj8utfsg8c0qr6v74"
+    }
+  ],
+  "pagination": {
+    "next_key": null,
+    "total": "7"
+  }
+}
+```
```

Comment:
I don't think the example output needs to be too long. Two or three, like Chunk's example, should be sufficient.

Reply:
applied the feedback in [here](https://github.com/b-harvest/Canto/pull/15/commits/14beb18a4543dc2e64b82b42e1be43b58daa201b)